src/mots/directory.py

```diff
@@ -129,7 +129,11 @@ def peers_and_owners(self):
         peers_and_owners = set()
         for module in modules_and_submodules:
             peers_and_owners.update(
-                [person["bmo_id"] for person in module.peers + module.owners]
+                [
+                    person["bmo_id"]
+                    for person in module.peers + module.owners
+                    if "bmo_id" in person
+                ]
```

Comment:
This essentially means that a peer/owner without a BMO ID wouldn't be returned by `Directory.peers_and_owners`, correct? Maybe we could document that in the docstring, or somehow indicate that this is returning only peers/owners that are on BMO. Is there a way to get all peers/owners that doesn't require a `bmo_id`?

Reply:
Yes, this is the same behaviour in the CLI today, and was discussed in the original ask. I'll add a comment. Currently there is only one "alias" (and adding any others would require modifying enforcement mechanisms). The main way this method is used is to link things back to an actual person (i.e., bugzilla ID), so I don't foresee, at least in the current way that aliases are implemented, the need for this to include aliases as well.